packages/manager/apps/dedicated/client/app/dedicated/server/servers.html

```diff
@@ -4,7 +4,7 @@
     ></oui-header>
 
     <ovh-manager-auto-renew-2016-deployment-banner
-        data-ng-if="$ctrl.isAutorenew2016DeploymentBannerAvailable"
+        data-show="$ctrl.isAutorenew2016DeploymentBannerAvailable"
```

Comment:
**question:** Any particular reason why `ng-if` has been changed to `show`?
It is same functionality no?

Reply:
I had to do it in order to use it in  vpsHeader view.
Vps listing page don't have dedicated controller because it use the layout helper. So I can't use ng-if. This is why I change to use a "show" binding and bind directly the resolve into it in the "vpsHeader" view section : packages/manager/modules/vps/src/vps.routing.js:11